querydsl-sql-codegen/src/main/java/com/querydsl/sql/codegen/support/SQLTypeMapping.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.querydsl.sql.codegen.support;
+
+import com.querydsl.sql.Configuration;
+
+/**
+ * {@code SQLTypeMapping} customizes mappings SQL symbolic type to data types.
+ *
+ * @author Walery Wysotsky
+ *
+ */
+public class SQLTypeMapping implements Mapping {
```

Comment:
should this be TypeNameMapping instead? the SQL prefix feels redundant

Reply:
"SQLTypeMapping" is used because it's mapping from SQL field types to Java. On my mind "TypeNameMapping" is more global and not describe what exactly we map - java to SQL or SQL to java... But if you need to change class name I will not mind